src/hooks/useHtmlPaste/index.ts

```diff
@@ -0,0 +1,129 @@
+import {useNavigation} from '@react-navigation/native';
+import ExpensiMark from 'expensify-common/lib/ExpensiMark';
+import {useCallback, useEffect} from 'react';
+import type UseHtmlPaste from './types';
+
+const useHtmlPaste: UseHtmlPaste = (textInputRef, checkComposerVisibility, isUnmountedOnBlur = true) => {
```

Comment:
`isUnmountedOnBlur` is a strange variable name to me. I am not sure if I would know when to use that or not based on the name. I get that we are trying to say that the Composer will basically never unmount and for that reason we need to add or remove the listeners. But think we should name this for what will happen if it's passed.

e.g. `updateListenersOnNavigation` or something

But actually - would it be so bad if we just had the same listeners for the private notes page? Maybe we can just attach/remove them anyway to make this less confusing?

Reply:
changed it to `unregisterListenerOnBlur`

Reply:
**compromise:** `unregisterListenerOnScreenBlur` ?

This way it's clear that it's the screen event and not the text inputs that we are watching.